cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -92,13 +93,33 @@ def plot(self, ax: Optional[plt.Axes] = None, **plot_kwargs: Any) -> plt.Axes:
             fig, ax = plt.subplots(1, 1, figsize=(8, 8))  # pragma: no cover
             ax = cast(plt.Axes, ax)  # pragma: no cover
         ax.set_ylim((0.0, 1.0))  # pragma: no cover
-        ax.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro-', **plot_kwargs)
+        ax.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro', label='data', **plot_kwargs)
+        x = np.linspace(self._num_cfds_seq[0], self._num_cfds_seq[-1], 100)
+        fit = self._fit_exponential()
+        ax.plot(x, fit[0][0] * fit[0][2] ** x + fit[0][1], '--k', label='fit')
+        ax.legend(loc='upper right')
         ax.set_xlabel(r"Number of Cliffords")
         ax.set_ylabel('Ground State Probability')
         if show_plot:
             fig.show()
         return ax
 
+    def pauli_error(self) -> float:
+        """Returns the Pauli error inferred from randomized benchmarking."""
+        fit = self._fit_exponential()
```

Comment:
nit: 
```py3
opt_params, *_ = self._fit_exponential()
p = opt_params[2]
```

Reply:
What is the difference between `*_` and `_`?

Reply:
`_` is a variable, `*_` is an iterable. so `opt_params, _ = self._fit_exponential()` will fail because it will try to unpack more than 2 objects into just 2. but `opt_params, *_ = self._fit_exponential()` will succeed because it will match the first object to `opt_params` and the rest will be given as a list to `_`

Reply:
Except in this case, `_` should work because `scipy.optimize.curve_fit` has two outputs, no?

Reply:
I see this after call it, but the documentation lists other values https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.curve_fit.html#scipy-optimize-curve-fit

Reply:
Yeah, the other outputs are only if `full_output == True`